ideascube/mediacenter/tests/test_remove_medias.py

```diff
@@ -0,0 +1,100 @@
+import os, stat
+
+from django.core.management import call_command
+
+import pytest
+
+from ideascube.mediacenter.models import Document
+
+pytestmark = pytest.mark.django_db
+CSV_PATH = os.path.join(os.path.dirname(__file__), 'data/metadata.csv')
+
+
+def write_metadata(metadata):
+    with open(CSV_PATH, 'w') as f:
+        f.write(metadata)
+
+
+def teardown_function(function):
+    if os.path.exists(CSV_PATH):
+        os.remove(CSV_PATH)
+
+
+def test_clean_orphans_should_not_fail_if_no_media_or_file():
+    call_command('remove_medias', 'clean_orphans')
+
+
+def test_clean_orphans_should_not_remove_files_associated_with_documents(settings):
+    metadata = ('title,summary,credits,path,preview\n'
+                'my video,my video summary,BSF,a-video.mp4,an-image.jpg\n')
+    write_metadata(metadata)
+    call_command('import_medias', CSV_PATH)
```

Comment:
We're sort of moving away from the `import_medias` workflow, so eventually, some time in the (hopefully not too) distant future we might end up completely removing that command.

So maybe we shouldn't add new things which rely on it?